app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -576,6 +594,10 @@ def humanized_display_type
       self.class.supported_display_types[display_type]
     end
 
+    def associated_host(vm)
```

Comment:
from some reason this shows as it was moved?

Reply:
I messed up the rebase. Fixed now.